themes/material/default/selectidp-links.php

```diff
@@ -120,4 +184,23 @@ function clickedAnyway(idpName) {
     <?php include __DIR__ . '/../common-footer.php' ?>
 </div>
 </body>
+<style>
+    .container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+    .button-container {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+    .mdl-card {
+        border-radius: 8px;"
```

Comment:
This looks like a typo:
```suggestion
        border-radius: 8px;
```